openmdao/core/driver.py

```diff
@@ -152,7 +164,7 @@ def __init__(self, **kwargs):
         self.supports.declare('linear_constraints', types=bool, default=False)
         self.supports.declare('two_sided_constraints', types=bool, default=False)
         self.supports.declare('multiple_objectives', types=bool, default=False)
-        self.supports.declare('integer_design_vars', types=bool, default=False)
+        self.supports.declare('integer_design_vars', types=bool, default=True)
```

Comment:
Just curious why support for integer design vars is True by default.

Reply:
There were a bunch of teats with Driver failing when I put the "supports" check in, but I fixed something else in the meantime, and now they pass with False supports, so I will change it back to False.